### DISCUSSION TITLE
Load balancing with upstream or other alternative solutions in nginx proxy manager

### DISCUSSION THREAD

**Discussion by MrPhyaeSoneThwim** [NginxProxyManager/nginx-proxy-manager#2670] [Q&A]
How can I implement the loading balancing with upstream for following example in nginx proxy manager. Because I didn't see any UI interface for upstream setup in the proxy manager. If the proxy manager doesn't support, Is there any alternative ways to implement the loading balancing. 

```
http {
    upstream backend {
        server 10.0.0.1:1337;
        server 10.0.0.2:1337;
    }
 
    server {
        listen 80;
        server_name example.com;
 
        location / {
            proxy_pass http://backend;
            proxy_set_header Host $host;
            proxy_set_header X-Real-IP $remote_addr;
        }
    }
}
```

**Comment by juniorcsa2022** [comment]
Como posso implementar o balanceamento de carga com o upstream para seguir o exemplo no gerenciador de proxy nginx. Porque não vi nenhuma interface UI para configuração upstream no gerenciador de proxy. Se o gerenciador de proxy não suportar, existe alguma forma alternativa de implementar o balanceamento de carga.

http {
    upstream backend {
        server 10.0.0.1:1337;
        server 10.0.0.2:1337;
    }
 
    server {
        listen 80;
        server_name example.com;
 
        location / {
            proxy_pass http://backend;
            proxy_set_header Host $host;
            proxy_set_header X-Real-IP $remote_addr;
        }
    }
}
